pages/1.10/administering-clusters/add-a-node/index.md

```diff
@@ -81,5 +81,5 @@ Copy the archived DC/OS installer file (`dcos-install.tar`) to the agent node. T
         dcos node --json | jq --raw-output '.[] | select(.reserved_resources.slave_public != null) | .id' | wc -l
         ```
 
- [1]: /1.10/installing/oss/custom/gui/
- [2]: /1.10/installing/oss/custom/cli/
+ [1]: /1.10/installing/gui/
```

Comment:
This is not right

Reply:
Remove this link/reference since GUI based installation method is deprecated.